gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
I'm actually -1 to that one -- we're already seeing some weird version numbers, which are really versions + revisions? Seems like just saying "this is the name of the GCD binary" (`gcd-v1beta2-revX-y.z.a`) is enough, right? I can't see ever using `'gcd'` separately from the `'v1-beta2-rev....'` string...

@mziccard - thoughts?